control-plane-operator/controllers/hostedcontrolplane/assets/olm/0000_50_olm_00-namespace.yaml.notneeded

```diff
@@ -1,26 +0,0 @@
-apiVersion: v1
```

Comment:
is this commit undoing changes introduced by a previous commit? Please let's consolidate those and squash them in one with final outcome.